### PR TITLE
feat: Handle zero max cap

### LINKED ISSUE #100: Virtual staking zero max-cap behaviour
Verify with the Cosmos SDK that zero max-cap means / implies **no bonded amounts**.

If that's not the case, send `Unbond` messages for each bonded tokens in the case max-cap is zero.

### PR DESCRIPTION
**Pull request by trinitys7**
Close #100 
Due to virtual staking and converter contract does not have delegator info, we need to resend the msg to external staking contract.

After external staking is called via ibc, converter contract get the Ack message, if it success, a msg to virtual staking contract will be executed to unbond the given amount by delegator.

### CODE REVIEW

**Comment by JakeHartnell** [comment]
Needs `cargo fmt` to fix linting errors.

File: packages/apis/src/ibc/packet.rs [hunk 1]
```diff
@@ -110,6 +112,14 @@ pub enum ConsumerPacket {
         /// This has precedence over all other events in the same packet.
         slashed: Vec<ValidatorSlashInfo>,
     },
+    InternalUnstake {
```
Comment (JakeHartnell) [inline packages/apis/src/ibc/packet.rs:115]: Please add a comment about this method for generated docs. Future devs will thank you. ❤️

File: packages/apis/src/virtual_staking_api.rs [hunk 2]
```diff
@@ -51,6 +53,17 @@ pub trait VirtualStakingApi {
         amount: Coin,
     ) -> Result<Response<Self::ExecC>, Self::Error>;
 
+    /// TODO: docs for this function
+    ///
```
Comment (JakeHartnell) [inline packages/apis/src/virtual_staking_api.rs:57]: TODO : )

File: packages/bindings/src/msg.rs [hunk 3]
```diff
@@ -30,6 +30,9 @@ pub enum VirtualStakeMsg {
     /// It will then burn those tokens from the caller's account,
     /// and update the currently minted amount.
     Unbond { amount: Coin, validator: String },
+    /// TODO: Add docs for this msg
+    ///
```
Comment (JakeHartnell) [inline packages/bindings/src/msg.rs:34]: TODO again : )

**Review by JakeHartnell — COMMENTED** [review]
LGTM, just a few comments mostly around documentation and cleaning up TODOs.

**Review by vuong177 — APPROVED** [review]

**Pull request merged**